dicegame/runner.py

```diff
@@ -2,54 +2,78 @@
 from .utils import i_just_throw_an_exception
 
 class GameRunner:
+    """
+    Hosts games of Sum the Dice.
+
+    Attributes:
+        round: An integer indicating the current round.
+        wins: An integer indicating how many times the user has won in total.
+        losses: An integer indicating how many times the user has lost in total.
+        c: An integer indicating the number of consecutive wins the user has
+           right now.
+    """
+
+    # Constants
+    N_DICE = 5          # Number of dice to roll

```

Comment:
This is okay as a constant, but maybe it would be better to have a global variable?